src/main/java/db/migration/V3__Update_table.java

```diff
@@ -0,0 +1,30 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * Example of a Java-based migration.
+ */
+public class V3__Update_table implements JdbcMigration {
+    private static Logger LOGGER = LoggerFactory.getLogger(V3__Update_table.class);
+
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        PreparedStatement statement =
+                connection.prepareStatement("INSERT INTO TEST (name) VALUES ('test3')");
+
+        try {
+            statement.execute();
+        } catch (SQLException e) {
+            LOGGER.error("Migration failed", e);
+        } finally {
+            statement.close();
```

Comment:
Instead of closing manually I think there are possible to use try-with-resourses:

```
try (PreparedStatement statement = connection.prepareStatement(insertSQL)) {
   // work with statement
} catch (SQLException e) {
  // handle error
}

// java closed statement properly for you